### PR TITLE
chore(dependency): update jquery and select2 version

### LINKED ISSUE #1946: Update dependencies having security vulnerabilities
### Description

- Currently fossology is using jQuery version 3.3.1(released on 20 Jan 2018) which has 3 security vulnerabilities 
upgrade to latest version of 3.6.0 (released on 2 march 2021)
Reference: https://snyk.io/vuln/npm:jquery?lh=3.3.1

- Currently fossology is using select2 version 4.0.5 (released on 26 oct 2017) which has 1 security vulnerability
upgrade to latest stable version of 4.0.13 (released on 28 Jan 2020)
Reference: https://snyk.io/vuln/npm:select2?lh=4.0.5

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Description

Updated dependencies having security vulnerabilities. 

Closes #1946  

### Changes

- Updated jQuery version from 3.3.1 to 3.6.0
- Updated select2 version from 4.0.5 to 4.0.13

### CODE REVIEW

**Comment by GMishx** [comment]
Hi @Aman-Codes , can you please add `install/db/db.cron` and `install/fossdash/fossdash-publish.py` to `.gitignore` so they do not end up in commits by mistake?

**Comment by Aman-Codes** [comment]
> Hi @Aman-Codes , can you please add `install/db/db.cron` and `install/fossdash/fossdash-publish.py` to `.gitignore` so they do not end up in commits by mistake?

Updated the .gitignore file

**Review by GMishx — APPROVED** [review]
Changes looks good. Needs test.

**Comment by GMishx** [comment]
https://github.com/fossology/fossology/blob/03c27e1183af2cfd99b9ff927648ad1aa9b47c64/src/www/ui/template/ui-browse.js.twig#L65
Please replace `fnDraw()` with `draw()`.

File: src/www/ui/template/ui-browse.js.twig [hunk 1]
```diff
@@ -62,7 +62,7 @@ $(document).ready(function() {
   browseTable = createBrowseTable();
   $('.clickable-folder').click(function(event){
     folder = $(this).attr('data-folder');
-    browseTable.fnDraw();
+    browseTable.Draw();
```
Comment (GMishx) [inline src/www/ui/template/ui-browse.js.twig:65]: I am sorry, it is supposed to be `draw()`
https://datatables.net/reference/api/draw()

**Review by GMishx — CHANGES_REQUESTED** [review]

**Review by GMishx — APPROVED** [review]
Tested, working fine.

**Pull request merged**